opm/parser/eclipse/EclipseState/InitConfig/Equil.cpp

```diff
@@ -0,0 +1,79 @@
+#include <opm/parser/eclipse/Deck/DeckItem.hpp>
+#include <opm/parser/eclipse/Deck/DeckKeyword.hpp>
+#include <opm/parser/eclipse/Deck/DeckRecord.hpp>
+#include <opm/parser/eclipse/EclipseState/InitConfig/Equil.hpp>
+
+namespace Opm {
+
+    EquilRecord::EquilRecord( const DeckRecord& record ) :
+            datum_depth( record.getItem( 0 ).getSIDouble( 0 ) ),
+            datum_depth_ps( record.getItem( 1 ).getSIDouble( 0 ) ),
+            water_oil_contact_depth( record.getItem( 2 ).getSIDouble( 0 ) ),
+            water_oil_contact_capillary_pressure( record.getItem( 3 ).getSIDouble( 0 ) ),
+            gas_oil_contact_depth( record.getItem( 4 ).getSIDouble( 0 ) ),
+            gas_oil_contact_capillary_pressure( record.getItem( 5 ).getSIDouble( 0 ) ),
+            live_oil_init_proc( record.getItem( 6 ).get< int >( 0 ) ),
```

Comment:
Observe that the three `int`values at the end of record are really used as enums:

```C++
if (live_oil_init_proc > 0)
   // Use init method1
else
   // Use init method2
```

What data is internalized is of course an implementation detail, but the public API should focus on the actually interesting data:

```C++ 
enum LiveOiilInitMethod {RSVD , CONSTANT_RS}

enum LiveOilInitMethod EquilRecord::liveOilInitProcedure() const {
     if (live_oil_init_proc > 0)
        return RSVD;
     else
        return CONSTANT_RS;
}
```

and similar for the two last items.